hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RemoteParam.java

```diff
@@ -77,4 +77,4 @@ public String toString() {
         .append(")")
         .toString();
   }
-}
\ No newline at end of file
+}
```

Comment:
Avoid

Reply:
Sorry, I edited it directly on the Github page and did not take any action on the last line, but it automatically has+} . If this impact is necessary, I can resubmit it for a try.